db/migrate/20191213094311_add_document_ref_to_whitehall_imports.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddDocumentRefToWhitehallImports < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :whitehall_imports,
+                  :document,
+                  foreign_key: { on_delete: :restrict },
+                  null: true
```

Comment:
May want index: false here as I don't expect we'll be using the index.

Reply:
Sure!